packages/components/src/ConnectionCard/ConnectionCard.js

```diff
@@ -50,18 +53,24 @@ export default function ConnectionCard({
   padding = true,
   wayForth,
   wayBack,
-  // localizedPrice,
+  price,
   duration,
   badges,
 }: Props) {
+  const localizedPrice = price =>
+    price &&
+    new Intl.NumberFormat(price.locale, {
```

Comment:
We will need to create a Price component in shared that will render LocalizedPrice under the hood, and it will deal with the localization part.

Reply:
Besides, I don't know how this works on Android, I will need to test the app there (there usually are problems when it comes to Intl support, though it might have been fixed in a recent release of RN)

Reply:
Sure, but this PR does not include the patch, so merging it into `master` would break on Android. I will create a PR to add the patch of the Intl library, and then your PR should work nicely. 

![screenshot_1547458823](https://user-images.githubusercontent.com/22741774/51105278-d8e13480-17e8-11e9-83e3-9821aa482c0d.png)

Reply:
Done